### PR TITLE
Add more math functions

### PR DESCRIPTION
**Pull request by navis**
Adding methods in java.lang.Math class as function.

### CODE REVIEW

File: common/src/main/java/io/druid/math/expr/Function.java [hunk 1]
```diff
@@ -26,39 +26,608 @@
...
+      Number x = args.get(0).eval(bindings);
+      if (x instanceof Long) {
+        return x.longValue() > 0 ? args.get(1).eval(bindings) : args.get(2).eval(bindings);
+      } else {
+        return x.doubleValue() > 0 ? args.get(1).eval(bindings) : args.get(2).eval(bindings);
+      }
     }
   }
-}
+}
```
Comment (himanshug) [inline common/src/main/java/io/druid/math/expr/Function.java]: newline

**Comment by himanshug** [comment]
:+1:

File: common/src/main/java/io/druid/math/expr/Function.java (hunk 1 above)
Comment (navis) [inline common/src/main/java/io/druid/math/expr/Function.java]: done

**Comment by fjy** [comment]
@navis can we get some documentation about using these math functions?

**Comment by navis** [comment]
@fjy I'll append function list to math-expr.md

**Comment by fjy** [comment]
👍

**Pull request merged**